README.md

```diff
@@ -113,9 +113,7 @@ that is where `setup.sh` will install hax.
 
 ## Supported Subset of the Rust Language
 
-Hax intends to support full Rust, with the two following exceptions, promoting a functional style:
- 1. no `unsafe` code (see https://github.com/hacspec/hax/issues/417);
- 2. mutable references (aka `&mut T`) on return types or when aliasing (see https://github.com/hacspec/hax/issues/420).
+Hax intends to support full Rust, with the one exception, promoting a functional style: mutable references (aka `&mut T`) on return types or when aliasing (see https://github.com/hacspec/hax/issues/420) are forbidden.
```

Comment:
We may have to change this as well. But dropping the `unsafe` is definitely a good call.